File: yongkeun/3week/아이스아메리카노.js
function solution(money) {
  return [parseInt(money/5500), money%5500] //배열의 첫번째에는 5500을 나눈 정수 값을 넣고, 두번째에는 아메리카노 가격5500의 나머지를 넣는다
}

File: minwook/3주차/아이스_아메리카노.java
class Solution {
    public int[] solution(int money) {
        int[] answer = new int[2];
        answer[0] = money / 5500; // 아이스 아메리카노의 잔 수
        answer[1] = money - answer[0] * 5500; // 남은 돈 money % answer[0]도 가능
        return answer; // 잔 수와 남은 돈 배열로 출력
    }
}